### ISSUE TITLE
Accessibility statement for Design System

### ISSUE THREAD

**Issue by martinwake** [dwp/design-system#349]
## What
Publish an accessibility statement for the Design System website following the [standard template for public sector websites](https://www.gov.uk/government/publications/sample-accessibility-statement/sample-accessibility-statement-for-a-fictional-public-sector-website).

## Why
Accessibility requirement

## Done when
- [x] Compliance status agreed
- [x] Any problems documented
- [x] Accessibility statement published - https://design-system.dwp.gov.uk/accessibility
- [ ] Date for review set

## Outcomes
- [x] Accessibility statement published

## Who needs to know about this
Design System Team
Accessibility team

**Comment by martinwake** [comment]
Draft content (based on template):

# Accessibility statement for the DWP Design System
This accessibility statement applies to the DWP Design System website at www.design-system.dwp.gov.uk.

This website is run by the Department for Work and Pensions (DWP). We want as many people as possible to be able to use this website. For example, that means you should be able to:

- change colours, contrast levels and fonts
- zoom in up to 300% without the text spilling off the screen
- navigate most of the website using just a keyboard
- navigate most of the website using speech recognition software
- listen to most of the website using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)

We’ve also made the website text as simple as possible to understand.

[AbilityNet](https://mcmw.abilitynet.org.uk/) has advice on making your device easier to use if you have a disability.

## How accessible this website is
[Note: use this section to provide information that a disabled user can act on - for example, avoid a particular section of the website, or request an alternative version rather than waste time trying to make it work with their assistive technology. Try to list in order of most impact to least impact.]

We know some parts of this website are not fully accessible:

- the text will not reflow in a single column when you change the size of the browser window
- you cannot modify the line height or spacing of text
- most older PDF documents are not fully accessible to screen reader software
- live video streams do not have captions
- some of our online forms are difficult to navigate using just a keyboard
- you cannot skip to the main content when using a screen reader
- there’s a limit to how far you can magnify the map on our ‘contact us’ page

## Feedback and contact information
If you need information on this website in a different format like accessible PDF, large print, easy read, audio recording or braille:

email [email address]
call [phone number]
[add any other contact details]
We’ll consider your request and get back to you in [number] days.

If you cannot view the map on our ‘contact us’ page, call or email us [add link to contact details page] for directions.

## Reporting accessibility problems with this website
We’re always looking to improve the accessibility of this website. If you find any problems not listed on this page or think we’re not meeting accessibility requirements, contact: [provide both details of how to report these issues to your organisation, and contact details for the unit or person responsible for dealing with these reports].

## Enforcement procedure
The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018 (the ‘accessibility regulations’). If you’re not happy with how we respond to your complaint, [contact the Equality Advisory and Support Service (EASS)](https://www.equalityadvisoryservice.com/).

[Note: if your organisation is based in Northern Ireland, refer users who want to complain to the Equalities Commission for Northern Ireland (ECNI) instead of the EASS and EHRC.]

## Contacting us by phone or visiting us in person
We provide a text relay service for people who are D/deaf, hearing impaired or have a speech impediment.

Our offices have audio induction loops, or if you contact us before your visit we can arrange a British Sign Language (BSL) interpreter.

Find out how to contact us [add link to contact details page].

## Technical information about this website’s accessibility
[Note: this form of wording is legally required, so do not change it.]

[Name of organisation] is committed to making its website accessible, in accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018.

### Compliance status
[Note: say that the website is fully compliant if the website meets WCAG 2.1 AA standard in full. Say that it’s partially compliant if it meets most requirements of the WCAG 2.1 AA standard. If it does not meet most requirements of the WCAG 2.1 AA standard, say that it’s not compliant.

If your website is either partially compliant or not compliant WCAG 2.1 AA standard, you’ll need to explain why. This will be due to one or both of the following:

non-compliances - this means the content in question is in scope of the regulations, but there’s an accessibility problem with it
an exemption - this means the inaccessible content is out of scope of the regulations, or it’d be a disproportionate burden for you to make it accessible
There’s a legally required way of expressing the compliance status of your website, so do not change it. The 3 options are as follows:]

This website is fully compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard.

This website is partially compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard, due to [insert one of the following: ‘the non-compliances’, ‘the exemptions’ or ‘the non-compliances and exemptions’] listed below.

This website is not compliant with the [Web Content Accessibility Guidelines version 2.1](https://www.w3.org/TR/WCAG21/) AA standard. The [insert one of the following: ‘non-compliances’, ‘exemptions’ or ‘non-compliances and exemptions’] are listed below.

[Note: delete the options that do not apply.]

### Non-accessible content
[Note: if the website is fully compliant with the WCAG 2.1 AA standard, you can leave the ‘Non-accessible content’ section out.

Otherwise, do not change the ‘Non-accessible content’ heading or the ‘The content listed below is non-accessible for the following reasons’ sentence - they’re legally required.

Do not change the ‘Non-compliance with the accessibility regulations’, ‘Disproportionate burden’ and ‘Content that’s not within the scope of the accessibility regulations’ subheadings: they’re also legally required.

But if you need to list a lot of problems, you can break these subsections up with further subheadings - for example, ‘Navigation and accessing information’ or ‘Interactive tools and transactions’.]

The content listed below is non-accessible for the following reasons.

### Non-compliance with the accessibility regulations
[Note: In this subsection, list:

- accessibility problems
- which of the WCAG 2.1 AA success criteria the problem fails on
- when you plan to fix the problem

Do not include any problems where you’re claiming [disproportionate burden](http://www.legislation.gov.uk/uksi/2018/952/regulation/7/made), or where the problem is outside the scope of the accessibility regulations (those should go in the subsections below).]

Some images do not have a text alternative, so people using a screen reader cannot access the information. This fails WCAG 2.1 success criterion 1.1.1 (non-text content).

We plan to add text alternatives for all images by September 2020. When we publish new content we’ll make sure our use of images meets accessibility standards.

### Disproportionate burden
[Note: in this subsection list accessibility problems you’re claiming would be a disproportionate burden to fix

Bear in mind that something which is a disproportionate burden now will not necessarily be a disproportionate burden forever. If the circumstances change, your ability to claim disproportionate burden may change too.]

### Navigation and accessing information
There’s no way to skip the repeated content in the page header (for example, a ‘skip to main content’ option).

It’s not always possible to change the device orientation from horizontal to vertical without making it more difficult to view the content.

It’s not possible for users to change text size without some of the content overlapping.

### Interactive tools and transactions
Some of our interactive forms are difficult to navigate using a keyboard. For example, because some form controls are missing a ‘label’ tag.

Our forms are built and hosted through third party software and ‘skinned’ to look like our website.

We’ve assessed the cost of fixing the issues with navigation and accessing information, and with interactive tools and transactions. We believe that doing so now would be a disproportionate burden within the meaning of the accessibility regulations. We will make another assessment when the supplier contract is up for renewal, likely to be in [rough timing].

### Content that’s not within the scope of the accessibility regulations
[Note: in this subsection list accessibility problems that fall outside the scope of the accessibility regulations.]

#### PDFs and other documents
Some of our PDFs and Word documents are essential to providing our services. For example, we have PDFs with information on how users can access our services, and forms published as Word documents. By September 2020, we plan to either fix these or replace them with accessible HTML pages.

The accessibility regulations [do not require us to fix PDFs or other documents published before 23 September 2018](http://www.legislation.gov.uk/uksi/2018/952/regulation/4/made) if they’re not essential to providing our services. For example, we do not plan to fix [example of non-essential document].

Any new PDFs or Word documents we publish will meet accessibility standards.

#### Live video
We do not plan to add captions to live video streams because live video is [exempt from meeting the accessibility regulations](http://www.legislation.gov.uk/uksi/2018/952/regulation/4/made).

## What we’re doing to improve accessibility
[Note: publishing an accessibility roadmap is optional. It’s a good idea to publish one if you want to be specific about the order you’re planning to tackle accessibility issues, and there’s no space to do so in the accessibility statement itself.]

Our accessibility roadmap [add link to roadmap] shows how and when we plan to improve accessibility on this website.

## Preparation of this accessibility statement
[Note: the wording about when the statement was prepared is legally required, so do not change it.]

This statement was prepared on [date when it was first published]. It was last reviewed on [date when it was last reviewed].

This website was last tested on [date]. The test was carried out by [add name of organisation that carried out test, or indicate that you did your own testing].

We used this approach to deciding on a sample of pages to test [add link to explanation of how you decided which pages to test].

[Note: you do not have to use this approach to sampling, but you should link to a full explanation of what you tested and how you chose it. If you get a third party auditor to test your website for you, they should include sampling details in test report - so you can just to link to that.]

You can read the full accessibility test report [add link to report].

[Note: publishing the test report is optional, but doing so may allow you to make your accessibility statement shorter and more focused.]

**Comment by mitz-lad** [comment]
Updated page to use side navigation with links to sibling pages. @simoneduca @dtaylorbrown I updated the logic for the header and mobile navigation to accommodate for sections which may not have a landing page, could someone please do a quick peer review to make sure I haven't missed/broken anything?
